migrations/migrations.go

```diff
@@ -12,74 +12,47 @@ import (
 	"github.com/insolar/block-explorer/etl/models"
 )
 
+func createTables(tx *gorm.DB) error {
```

Comment:
We initial database tables with separate structs, not structs from `models` package, so if we change struct in models (add new field, update existed etc.), we write new migration for that change, so database was updated too.
If you want to reuse creation of table, I think its better get `Migrations()` slice and add to it new migration with data to create.

Reply:
done